src/Goerli-DssSpell.t.base.sol

```diff
@@ -857,7 +857,7 @@ contract GoerliDssSpellTestBase is Config, DSTest, DSMath {
 
         (,,,, uint256 dust) = vat.ilks(_ilk);
         dust /= RAY;
-        uint256 amount = 2 * dust * 10 ** uint256(token.decimals()) / (_isOSM ? getOSMPrice(pip) : uint256(DSValueAbstract(pip).read()));
+        uint256 amount = 4 * dust * 10 ** uint256(token.decimals()) / (_isOSM ? getOSMPrice(pip) : uint256(DSValueAbstract(pip).read()));
```

Comment:
GNO has some weird metrics (350% collateralization ratio, 100,000 min vault debt).
Using `2 * dust` here would cause the test to fail because the `vat` would be in an unsafe state.

We need and amount of gem that is greater than the highest collateralization ratio for this to work properly.